webroot/bulk-uploader-sample-images/README

```diff
@@ -0,0 +1,10 @@
+These images are for use with the Bulk Uploader Addon.
```

Comment:
If you make the filename `README.md` then you can use markdown (like `#` for headings and such).

Then when someone is viewing on github they can see the styled version similar to [the top level readme](https://github.com/geodesicsolutions-community/website/blob/main/README.md)